src/solver/mlir_common.cpp

```diff
@@ -60,6 +60,17 @@ const char* DTypeName(miopenDataType_t ty)
     MIOPEN_THROW(miopenStatusInternalError, "Value outside of datatype enum");
 }
 
+static std::string GetIsaName(const miopen::TargetProperties& target)
+{
+#if ROCM_FEATURE_TARGETID_OFF
```

Comment:
Adding `#include <miopen/rocm_features.hpp>` is necessary.

Reply:
Done in e8de8d3